fixtures/src/components/uncontrolled/decimal.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Decimal from 'carbon-react/lib/__experimental__/components/decimal';
+import { LogConsumer } from '../log';
+
+const UncontrolledDecimal = () => {
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = e => log(e, { method: 'onChange' });
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <div id='uncontrolled_decimal'>
+              <h1>Uncontrolled Decimal</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be the
```

Comment:
I think there should be an explanation about `e.target.value` containing `rawValue` and `formattedValue`